styleguide/component-select.scss

```diff
@@ -101,49 +92,29 @@ $layout-parent-bg-color: $purple-25;
   justify-content: flex-start;
 }
 
+.selected-info {
+  // we're explicitly overriding the color below
```

Comment:
Yeah! It'll still do that with the shorthand. Just gave it a whirl to confirm:
http://codepen.io/davidandrewfrey/pen/wWLxNq